src/utils.ts

```diff
@@ -522,6 +522,11 @@ export function pathToRegex(aPath: string): string {
         aPath = escapeRegexSpecialChars(fileUrlPrefix) + aPath;
     }
 
+    if (guid) {
+        // Add a guid to the regexp to make it unique, without modifying what it matches. This will allow us to add duplicated breakpoints using CDTP
+        aPath = aPath + `(?:${guid}){0}`;
```

Comment:
No idea what you mean, this does modify what it matches

Reply:
I guess to set a breakpoint for the same path multiple times, but what do we need to do that for?

Reply:
The regexp should match the same things as before.
Then we can add all the new breakpoints, and then remove all the old ones.

If we don't support 2 breakpoints in the same locations, that code will fail if you change the condition of a breakpoint